lib/github_export/command.rb

```diff
@@ -108,7 +108,12 @@ def assets_download
             if options[:force] || !File.exist?(dest)
               verbose "DL #{url}"
               FileUtils.mkdir_p(File.dirname(dest))
-              File.binwrite(dest, client.get(url))
+              # Assets might be downloaded from S3 so we use curl (or httpclient?) without auth info instead of `client` object
+              # File.binwrite(dest, client.get(url))
+              cmd = "curl -o #{dest} #{url}"
```

Comment:
To detect error, [-f](https://curl.haxx.se/docs/manpage.html#-f) option is required.

Reply:
@minimum2scp Thanks! I added it.